articles/azure/azs-how-configure-cli.md

```diff
@@ -179,5 +179,15 @@ az vm create --resource-group testRG --name testVM --image OpenLogic:CentOS:6.9:
 There are some known issues that you must be aware of when using CLI in Azure Stack:
 
 * The CLI interactive mode i.e the `az interactive` command is not yet supported in Azure Stack.
+
 * To get the list of virtual machine images available in Azure Stack, use the `az vm images list --all` command instead of the `az vm image list` command. Specifying the `--all` option makes sure that response returns only the images that are available in your Azure Stack environment. 
+<<<<<<< HEAD
```

Comment:
What's this?

Reply:
These were automatically generated by github when we were merging different branches - however if you review the latest version of the document it shouldn't be present